torchchat/model.py

```diff
@@ -919,6 +937,58 @@ def apply_rotary_emb(x: Tensor, freqs_cis: Tensor) -> Tensor:
     return x_out2.type_as(x)
```

Comment:
nit: Can move `apply_rotary_emb` so that it is sequentially after `hf_apply_rotary_emb`?

Mainly for keeping concepts together

Reply:
I'd like to keep the current structure, with all HF rotary embedding functions grouped together and all previous embedding functions in a separate section.